static/js/find_account.js

```diff
@@ -0,0 +1,42 @@
+function validateEmail(email) {
```

Comment:
Check other `js` files exists `static/js/portico` folder, understand how they works, syntax, logic etc. See how validations are applied to html element and how html element are updated, these will give you better idea on how you can improve/change code written in this file.